doc/parse/package.go

```diff
@@ -31,10 +33,29 @@ func NewPackageDoc(dir string) (*doc.Package, error) {
 }
 
 func setDocFuncsMap(pkgDoc *doc.Package) {
-	funcsMap = make(map[string]*doc.Func, len(pkgDoc.Funcs))
+	typeFuncsMap := getPkgTypesFunctions(pkgDoc)
+	funcsMap = make(map[string]*doc.Func, len(pkgDoc.Funcs)+len(typeFuncsMap))
+
 	for _, fn := range pkgDoc.Funcs {
 		funcsMap[fn.Name] = fn
 	}
+
+	for k, fn := range typeFuncsMap {
+		funcsMap[k] = fn
+	}
+}
+
+func getPkgTypesFunctions(pkgDoc *doc.Package) map[string]*doc.Func {
+	result := make(map[string]*doc.Func)
+	for _, t := range pkgDoc.Types {
+		for _, f := range t.Methods {
+			if f.Doc != "" {
+				result[fmt.Sprintf("%s.%s", t.Name, f.Name)] = f
```

Comment:
Oh i see now why you dropped the`*`.
For the future, some high-level description of your pull request would have been helpful.

You don't want to only add the function name if the doc string is present, you should always add it.
So remove line 52 :)